packages/graphql-config-utilities/src/config.ts

```diff
@@ -1,14 +1,72 @@
+import {existsSync} from 'fs';
 import {promisify} from 'util';
 import {GraphQLConfig, GraphQLProjectConfig} from 'graphql-config';
 
-import './augmentations';
 import './polyfills';
-import {resolvePathRelativeToConfig, resolveSchemaPath} from './utilities';
 
 // we need to use an import/require here because it does not force consumers to
 // enable esModuleInterop in tsconfig.json
 import glob = require('glob');
 
+export const defaultGraphQLProjectName = 'GraphQL';
+
+// temporary utility until `graphql-config` supports this function natively
+// see: https://github.com/prisma/graphql-config/pull/113
+export function resolvePathRelativeToConfig(
```

Comment:
i am merging the old `utilities.ts` into `config.ts` since they are essentially the same thing now.